src/EmailAddressValidator.php

```diff
@@ -0,0 +1,295 @@
+<?php
+
+namespace ipl\Validator;
+
+use ipl\I18n\Translation;
+
+/**
+ * Validates Email address
```

Comment:
Check the spelling. *Validates **an** email adress* or *Validates email address**es***